src/generators/networkCreator.js

```diff
@@ -80,6 +87,15 @@ export async function generateResourcesLocally(config) {
     copyDirectory(joinPath(libRootDir(), '7nodes'), configDir)
   }
 
+  if (isSplunk(config.network.splunk)) {
+    copyDirectory(joinPath(libRootDir(), 'splunk'), joinPath(configDir, 'splunk'))
+  }
+
+  if (config.network.txGenerate) {
+    copyDirectory(joinPath(libRootDir(), 'lib', 'scripts'), joinPath(configDir, 'scripts'))
+    copyFile(joinPath(networkPath, 'private_contract.js'), joinPath(configDir, 'scripts', 'private_contract.js'))
+  }
```

Comment:
generateResourcesLocally is only used for bash deployments, so you can get rid of this if it only works in docker-compose mode

Reply:
removed